Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -199,6 +200,11 @@ class TextureCacheBase
   TexPool::iterator FindMatchingTextureFromPool(const TCacheEntryConfig& config);
   TexAddrCache::iterator GetTexCacheIter(TCacheEntryBase* entry);
 
+  // Return all possible overlapping textures. As addr+size of the textures is not
+  // indexed, this may return false positives.
+  std::pair<TexAddrCache::iterator, TexAddrCache::iterator>
+  FindOverlappingTextures(u32 addr, u32 size_in_bytes);
```

Comment:
You are right, sorry, I misunderstood your original comment in thinking that you meant std::pair<const iter, const iter> but I see what you're saying now.  I didn't think about the implications of textures_by_address being const...drats.

And I do see how it would be difficult to convert things over to be const correct.  Sorry about that!